.github/workflows/benchmark.yml

```diff
@@ -54,9 +57,10 @@ jobs:
       uses: ./.github/actions/install-aiida-core
       with:
         python-version: '3.10'
+        from-lock: 'true'
 
     - name: Run benchmarks
-      run: pytest --db-backend psql --benchmark-only --benchmark-json benchmark.json tests/
+      run: uv run pytest --db-backend psql --benchmark-only --benchmark-json benchmark.json tests/
 
     - name: Store benchmark result
       uses: aiidateam/github-action-benchmark@v3
```

Comment:
@unkcpz can you open this PR from origin? It looks like the store step is failing, likely due to permission issue.

Reply:
Thanks for the fix btw! Definitely forgot a few spots.